src/pages/superadmin/statistics/index.tsx

```diff
@@ -36,6 +36,16 @@ interface StatisticsProps {
   mockHunter?: MockHunterMetrics;
 }
 
+const formatPercentage = (value?: number): string => {
```

Comment:
Please add this to helpers, and write a unit test for it. src/helpers/helpers-extended.ts

Reply:
Absolutely, I'll add the requested changes to helpers-extended.ts and ensure a unit test is written for it. I'll let you know as soon as it's completed.